lc-hax/Scripts/Modules/PhantomMod.cs

```diff
@@ -24,7 +24,8 @@ public sealed class PhantomMod : MonoBehaviour {
     }
 
     void Update() {
-        if (!Helper.CurrentCamera.IsNotNull(out Camera camera) || !camera.enabled) return;
+        if (!Helper.CurrentCamera.IsNotNull(out Camera camera)) return;
+        try { if (!camera.enabled) return; } catch { return; }
```

Comment:
For the sake of style, let's use the `Try` helper.

```suggestion
        if (!Helper.CurrentCamera.IsNotNull(out Camera camera) || Helper.Try(() => !camera.enabled)) return;
```